src/Pages/Dashboard/index.jsx

```diff
@@ -1,9 +1,21 @@
-import { FilterTrainingEvent } from "../../Parts";
-
+import { useContext } from "react";
+import { AppContext } from "../../Context";
+import {
+  FilterTrainingEvent,
+  MyTrainingCard,
+  AllTrainingEventTable,
+  MyTrainingEventTable,
+  AllTrainingEvent,
+  SectionHeader,
+} from "../../Parts";
 const Dashboard = () => {
+  const { view } = useContext(AppContext);
   return (
     <div>
+      <SectionHeader />
       <FilterTrainingEvent />
+      {view ? <MyTrainingCard /> : <MyTrainingEventTable />}
```

Comment:
Please wrap in one element or use react fragment!

Reply:
thankyou for review, this issue already fix in a7711523a75ebf1bee4167f348adc6eda02d0c44